UpdateVersion.csx

```diff
@@ -0,0 +1,29 @@
+////////////////////////////////////////////////////////////
+//
+// This is META-file for using in CI system.
+//                                         STAM, EpicMorg
+////////////////////////////////////////////////////////////
+
+using System;
+using System.IO;
+using System.Xml;
+
+var versionFilePath = "kasthack.binding.wf.csproj";
+var xmlDoc = new XmlDocument();
+xmlDoc.Load(versionFilePath);
+
+var versionNode = xmlDoc.SelectSingleNode("//Version");
+var currentVersion = versionNode.InnerText;
+
+Console.WriteLine($"Current version is: {currentVersion}");
+
+var buildNumber = Environment.GetEnvironmentVariable("GITHUB_RUN_NUMBER");
+var newVersion = $"{currentVersion.Split('.')[0]}.{currentVersion.Split('.')[1]}.{buildNumber}";
```

Comment:
So, you only keep the major release and set the build version to `{N}.{buildNumber}`, right? Why do you have the full version in the project file then?

Reply:
thank you. rewrited logic at all. resolved https://github.com/kasthack-labs/kasthack.binding.wf/pull/2/commits/659e8c905329ccda682cfc70a8f15ba35cfc83ff

Reply:
https://github.com/kasthack-labs/kasthack.binding.wf/pull/2/commits/7e92906ccca3e5a578a736cb9992c989c07c56bc